src/sparseml/core/recipe/recipe.py

```diff
@@ -240,6 +419,14 @@ def dict(self, *args, **kwargs) -> Dict[str, Any]:
 
 @dataclass
 class RecipeTuple:
+    """
+    A simple dataclass to hold a recipe, it's target_stages, and override_args
+
+    :param recipe: The Recipe instance to hold
+    :param target_stages: The target stages to use when simplifying the recipe
+    :param override_args: The override args to use when simplifying the recipe
```

Comment:
clarify target_stages are used to filter recipe stages, and override args override any args present in the recipe